tests/integration/integration_snowpark.py

```diff
@@ -0,0 +1,21 @@
+import json
+import pytest
+
+from snowflake.snowpark import Session
+
+
+@pytest.fixture(scope="session")
+def snowpark_session():
+    config = """{
+        "account": "",
+        "user": "",
+        "password": "",
+        "host": "",
+        "port": -1,
+        "protocol": "",
+        "role": "",
+        "database": "",
+        "schema": "",
+        "warehouse": ""
+    }"""
+    yield Session.builder.configs(json.loads(config)).create()
```

Comment:
Should we load the values from environment? Also it would be good to add note in `contributing.md` (?) how to run the integration tests

Reply:
Added task for it